ethstorage/storage_manager.go

```diff
@@ -153,6 +153,10 @@ func (s *StorageManager) Reset(newL1 int64) error {
 	return nil
 }
 
+func (s *StorageManager) GetLastKVIdxByBlockNumber(blockNumber int64) (uint64, error) {
```

Comment:
what do we need to do this change？

Reply:
I think it's ok as long as the behavior is documented